dbt/include/redshift/macros/relations/materialized_view/describe.sql

```diff
@@ -13,9 +13,9 @@
             tb.sortkey1,
             mv.autorefresh
         from svv_table_info tb
-        left join stv_mv_info mv
-            on mv.db_name = tb.database
-            and mv.schema = tb.schema
+        left join svv_mv_info mv
```

Comment:
To address @mikealfare's suggestion [here](https://github.com/dbt-labs/dbt-redshift/pull/647#issuecomment-1782202473).

@mikealfare did you want an explicit link to the GitHub issue as well? If so, you just add a code suggestion of what you are looking for?

```suggestion
        -- svv_mv_info is queryable by Redshift Serverless, but svt_mv_info is not
        left join svv_mv_info mv
```

Reply:
I'm an idiot, I just didn't read. @jiezhen-chen provided the rationale in the summary above. If I still get to be nit-picky, your suggested change has a typo; 'stv_mv_info` is misspelled. The comment addresses my concern though.

Reply:
Good eyes! Nit-picky is appreciated. I fixed the spelling above, and I'm going to commit this suggestion momentarily.